service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredComponentTest.java

```diff
@@ -63,7 +64,7 @@
     private DetectTranslationCompleteService detectTranslationCompleteService;
 
     @Test
-    public void When_NewStructuredTask_Expect_DatabaseUpdatedAndAddedToObjectStore() throws IOException {
+    public void When_NewStructuredTask0_Expect_DatabaseUpdatedAndAddedToObjectStore() throws IOException {
```

Comment:
NewStructuredTask0? Why the name change?

Reply:
haven't noticed it sorry.